fuel-tx/src/transaction.rs

```diff
@@ -1165,9 +1303,31 @@ mod tests {
             vec![[0u8; 64].to_vec().into()],
         );
         assert_eq!(
-            create_with_witnesses.metered_bytes_size(),
-            create_with_no_witnesses.metered_bytes_size()
-                + create_with_witnesses.witnesses.size_dynamic()
+            tx_with_witnesses.metered_bytes_size(),
+            tx_with_no_witnesses.metered_bytes_size()
+                + tx_with_witnesses.witnesses.size_dynamic()
+        );
+    }
+
+    #[test]
+    fn upload_metered_data_includes_witnesses() {
```

Comment:
This is just testing that the size of the tx bytes increases by 8 bytes + witness bytes size. I feel like that's what the test should be named.

Is there a way to inspect the bytes to show that they are equal except for the added witness + 8 bytes?

Reply:
Oh sorry. I meant, not just check that it's longer by `witness.size()` (I'm confused why `witness.size()` is equal to 64 + 8 (where does the 8 come from)).

I was saying that it might make sense to see that the witness bytes were added to the correct spot in the metered bytes. The test name kinda implies that it's checking that the bytes match.